docs/UserGuide.md

```diff
@@ -18,7 +24,7 @@ especially for those who are teaching multiple classes/modules at the same time.
 
 ## Quick start
 
-1. Ensure you have [Java `11`](https://www.oracle.com/java/technologies/downloads/#java11) or above installed in your Computer.
+1. Before getting started with TAPA, ensure you have [Java `11`](https://www.oracle.com/java/technologies/downloads/#java11) or above installed on your computer. You can check the version of Java on your computer by inputting `java -version` in your terminal.
```

Comment:
Maybe in this part, we can add and say like how to go to your terminal? I've seen some guides out there that actually specify this so maybe we can do it as well to make it as beginner-friendly as possible?

Reply:
Got it! Maybe I'll just link to an article that talks about that?